src/main/java/seedu/address/model/property/Property.java

```diff
@@ -0,0 +1,79 @@
+package seedu.address.model.property;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+import java.util.StringJoiner;
+
+import seedu.address.model.person.Address;
+
```

Comment:
Nice! Understandable that this is a Property class that contains all these related field so no need header comment if too tedious.